packages/css/src/unordered-list/unordered-list.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-unordered-list {
+  color: var(--amsterdam-unordered-list-color);
+  font-family: var(--amsterdam-unordered-list-font-family);
+  font-size: var(--amsterdam-unordered-list-font-size-narrow);
+  font-weight: var(--amsterdam-unordered-list-font-weight);
+  line-height: var(--amsterdam-unordered-list-line-height);
+  list-style: var(--amsterdam-unordered-list-list-style);
+  margin-block: var(--amsterdam-unordered-list-margin-block);
+  padding-inline-end: var(--amsterdam-unordered-list-padding-inline-end);
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-unordered-list-font-size-wide);
+  }
+}
+
+.amsterdam-unordered-list__item {
+  padding-inline-start: var(--amsterdam-unordered-list-item-padding-inline-start);
```

Comment:
This padding seems to be larger than the design

Reply:
If you change a UL margin and padding to 0 the bullet will be centered on the start of the window, I've taken the liberty to visualy put the list on the same position.